src/Build/CompatibilitySuppressions.xml

```diff
@@ -0,0 +1,77 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <Suppression>
+    <DiagnosticId>CP0002</DiagnosticId>
+    <Target>M:Microsoft.Build.Experimental.ProjectCache.ProjectCacheDescriptor.FromAssemblyPath(System.String,System.Collections.Generic.IReadOnlyCollection{Microsoft.Build.Graph.ProjectGraphEntryPoint},Microsoft.Build.Graph.ProjectGraph,System.Collections.Generic.IReadOnlyDictionary{System.String,System.String})</Target>
```

Comment:
Is there any way to make these more generic (like `M:Microsoft.Build.Experimental.ProjectCache*` or something)?

Reply:
Not really. You can either have a directly matching target or omit the target element entirely which will suppress the CP0002 diagnostic for all APIs in that left/right assembly tuple.